zipline/test_algorithms.py

```diff
@@ -784,6 +784,11 @@ def handle_data(context, data):
     pass
 """
 
+no_handle_data = """
+def initialize(context):
+    pass
```

Comment:
Do we really want this to be a valid strategy?  Should we require that you have handle_data OR a scheduled function?